nxpprog.py

```diff
@@ -968,15 +980,17 @@ def prepare_flash_sectors(self, start_sector, end_sector):
     def erase_sectors(self, start_sector, end_sector, verify=False):
         self.prepare_flash_sectors(start_sector, end_sector)
 
-        log("Erasing flash sectors %d-%d" % (start_sector, end_sector))
+        log("Erasing flash sectors %d-%d" % (start_sector, end_sector),
+            LogType.DEBUG)
```

Comment:
This is actually useful information for the user. This should be a normal log.

Reply:
Done.